mfRPMRules.mk

```diff
@@ -89,6 +89,12 @@ _spec_update:
 	sed -i 's#__buildarch__#$(Arch)#' $(PackagePath)/rpm/$(PackageName).spec
 	sed -i 's#__requires_list__#$(REQUIRED_PACKAGE_LIST)#' $(PackagePath)/rpm/$(PackageName).spec
 	sed -i 's#__build_requires_list__#$(BUILD_REQUIRED_PACKAGE_LIST)#' $(PackagePath)/rpm/$(PackageName).spec
+	sed -i 's#__is_arm__#$(IS_ARM)#' $(PackagePath)/rpm/$(PackageName).spec
+
+	if [ -e $(PackagePath)/scripts/postinstall.sh ]; then \
+		sed -i '\#\bpost\b#r $(PackagePath)/scripts/postinstall.sh' $(PackagePath)/rpm/$(PackageName).spec; \
```

Comment:
Does `scripts` get packaged?
Do these scriptlets get packaged (and if so, do they need to be)?
I'd prefer that the scriptlets live either in the base directory and get copied into `rpm` during `preprpm`, or live in `rpm`, and `cleanrpm` gets changed to not `rm -rf rpm`

Reply:
>Does scripts get packaged?
yes, according to generic rules
>Do these scriptlets get packaged (and if so, do they need to be)?
Yes, but I don't really see any harm in this. I guess they don't have to be packaged as their content is copied into `spec`.